packages/test-sync/src/commonTest/kotlin/io/realm/kotlin/test/mongodb/common/ApiKeyAuthTests.kt

```diff
@@ -49,7 +49,7 @@ class ApiKeyAuthTests {
 
     @BeforeTest
     fun setup() {
-        app = TestApp(appName = TEST_APP_PARTITION)
+        app = TestApp(this::class.simpleName, appName = TEST_APP_PARTITION)
```

Comment:
I guess not for this PR, but would be awfully nice if we could just inject the test name from a rule or something like that. This seems like a really annoying thing to have to always write ... but I guess less annoying than trying to debug without it 😅